qiskit/primitives/estimator.py

```diff
@@ -51,6 +47,7 @@ class Estimator(BaseEstimator[PrimitiveJob[EstimatorResult]]):
           this option is ignored.
     """
 
+    @deprecate_func(since="0.46.0", additional_msg="Use StatevectorEstimator instead.")
```

Comment:
In [the estimator V2 PR](https://github.com/Qiskit/qiskit/pull/11227/files#diff-18945a0dc9b0f336359e1a96ed462e05f045f7d83c8b24c801dac5cf1d3eac46R70), it looks like the new class is also called `Estimator`, right? not `StatevectorEstimator`. But it can be told apart from the V1 estimator by the import path.